src/components/ChecNavigation.vue

```diff
@@ -155,15 +155,19 @@ export default {
     controlled: Boolean,
   },
   data() {
-    const compiledTree = this.tree.map(this.parseSection);
     return {
       showDeepNav: false,
-      compiledTree,
-      activeSection: compiledTree[0],
+      currentActiveSection: null,
       listItemFocus: null,
     };
   },
   computed: {
+    compiledTree() {
+      return this.tree.map(this.parseSection);
+    },
+    activeSection() {
+      return this.currentActiveSection || this.compiledTree[0];
```

Comment:
This computed var is being assigned on line 265. You'll need to convert this to the `get`/`set` style of computed attribute or change line 265 to update `currentActiveSection` instead.